### ISSUE TITLE
Youtube video links show preview/embed

### ISSUE THREAD

**Issue by rufuspollock** [flowershow/flowershow#68]
Video URL links such as `https://www.youtube.com/watch?v=z2uAg-AIs-Y` (Youtube, etc) surrounded by newlines in markdown will embed as iframe or similar on the page.

Similar concept from: https://github.com/agentofuser/remark-oembed

Existing work in web3: https://github.com/life-itself/web3/issues/88 and PR https://github.com/life-itself/web3/pull/98

## Acceptance

* [x] A link like `https://www.youtube.com/watch?v=z2uAg-AIs-Y`
* [x] Display is responsive (not raw iframe)

**Comment by rufuspollock** [comment]
@khalilcodes suggest re-assign to @PhilippeduPreez for the next sprint.

**Comment by PhilippeduPreez** [comment]
FIXED [here](https://github.com/flowershow/flowershow/pull/197/commits/6904bcb4a614da9cef2f40624fece61d5726c1c2)

**Comment by rufuspollock** [comment]
@PhilippeduPreez 

- please do update acceptance items with some detail of what happened. i'd want much more info in the acceptance updates about what you did exactly.
- In general, can you summarize in the issue about the approach you took. (usually this would even happy early on in the work rather than at the end)

**Comment by PhilippeduPreez** [comment]
I Tested whether any existing packages worked, but I found them to be full of errors, so made a new plugin from scratch [here](https://github.com/flowershow/remark-embed-plus) and uploaded to npm.

**Comment by rufuspollock** [comment]
@PhilippeduPreez do we have documentation of this in flowershow e.g. https://flowershow.app/docs/social-previews - if not can we quickly add this with an example and/or screenshot of that.

**Comment by PhilippeduPreez** [comment]
docs added at https://flowershow.app/docs/social-previews
